src/components/CreateCoursePage/index.jsx

```diff
@@ -64,6 +64,7 @@ class CreateCoursePage extends React.Component {
         start: options.start,
         end: options.end,
         pacing_type: options.pacing_type,
+        rerun: '',
```

Comment:
You should be able to get away with not supplying this at all in the course creation page by just having the backend do a `request.data.get('rerun', '')` (which will default to `''`)